<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCAnalysisQuery</class>
 <widget class="QWidget" name="QCAnalysisQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1493</width>
    <height>944</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3,3,3,3,1">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>查询条件</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>时间范围:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>起始日期</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startDateBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>120</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>点击设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>~</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="endDateBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>120</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>点击设置</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>X轴:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CusLineEdit" name="XCount" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="5,10,1,10">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>     Y轴:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CusLineEdit" name="YNumberStart" native="true"/>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>~</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CusLineEdit" name="YNumberEnd" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,4">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>Y轴:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CusLineEdit" name="YCount" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="resetBtn">
       <property name="minimumSize">
        <size>
         <width>145</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnNomal.png);
padding: -1;}
QPushButton:pressed{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnClicked.png);
padding: -1;}
</string>
       </property>
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ConfirmBtn">
       <property name="minimumSize">
        <size>
         <width>145</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnNomal.png);
padding: -1;}
QPushButton:pressed{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnClicked.png);
padding: -1;}
</string>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="returnBtn">
       <property name="minimumSize">
        <size>
         <width>145</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnNomal.png);
padding: -1;}
QPushButton:pressed{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnClicked.png);
padding: -1;}
</string>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CusLineEdit</class>
   <extends>QWidget</extends>
   <header location="global">CusLineEdit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
